### PR TITLE
Implemented template for our sections

### PR DESCRIPTION
**Pull request by AnnaIstochnikova**
Create template for section, add styles for each screen version

### CODE REVIEW

File: src/components/templates/Section/Section.module.css
Comment (TN4HPEL3) [inline src/components/templates/Section/Section.module.css:1]: For example implement 2 or 3 section on on site and check if vertical scroll is not appearing 💡

File: src/style/variables.css
Comment (TN4HPEL3) [inline src/style/variables.css:1]: Variables in another branch/PR

**Review by TN4HPEL3 — CHANGES_REQUESTED** [review]

File: src/components/templates/Section/Section.module.css
Comment (AnnaIstochnikova) [reply src/components/templates/Section/Section.module.css:1]: Fixed this.

File: src/style/variables.css
Comment (AnnaIstochnikova) [reply src/style/variables.css:1]: I've created new branch.

**Review by TN4HPEL3 — APPROVED** [review]

**Pull request merged**